src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -114,13 +114,9 @@ class OTARequestor : public OTARequestorInterface, public BDXDownloader::StateDe
         mProviderEndpointId  = endpointId;
     }
 
-    // Application directs the Requestor to abort the download in progress. All the Requestor state (such
-    // as the QueryImageResponse content) is preserved
-    void AbortImageUpdate();
-
-    // Application directs the Requestor to abort the download in progress. All the Requestor state is
-    // cleared, UploadState is reset to Idle
-    void AbortAndResetState();
+    // Application directs the Requestor to cancel image update in progress. All the Requestor state is
+    // cleared, UpdateState is reset to Idle
+    void CancelImageUpdate();
```

Comment:
This API should be in OTARequestorInterface.h as well

Reply:
Fixed